base/src/main/java/io/spine/base/Identifier.java

```diff
@@ -510,4 +532,47 @@ Message fromMessage(Message message) {
             return result;
         }
     }
+
+    /**
+     * A message containing a single string field with the {@linkplain #FIELD_NAME name}.
+     */
+    private static class UuidMessage<I extends Message> {
```

Comment:
Please convert to a package-private class at the top level. The reason is that `Identifier` class is long enough already.